GitHubAssignees.ps1

```diff
@@ -379,7 +389,7 @@ function Remove-GithubAssignee
         'assignees' = $Assignee
     }
 
-    if ($PSCmdlet.ShouldProcess($Assignee -join ', ', "Remove assignee(s)"))
+    if ($Force -or $PSCmdlet.ShouldProcess($Assignee -join ', ', "Remove assignee(s)"))
```

Comment:
This should be:

```powershell
if ($Force -and -not $Confirm){
    $ConfirmPreference = 'None'
}

if ($PSCmdlet.ShouldProcess($Assignee -join ', ', "Remove assignee(s)"))
```

or else the `Whatif` parameter will be ignored. See: [Powershell: Everything you wanted to know about ShouldProcess](https://powershellexplained.com/2020-03-15-Powershell-shouldprocess-whatif-confirm-shouldcontinue-everything/#implementing--force).